metpy/calc/basic.py

```diff
@@ -88,6 +92,14 @@ def wind_direction(u, v):
     origshape = wdir.shape
     wdir = atleast_1d(wdir)
     wdir[wdir <= 0] += 360. * units.deg
+
+    # Handle oceanographic convection
+    if convention == 'to':
+        wdir -= 180 * units.deg
+        wdir[wdir <= 0] += 360. * units.deg
+    elif convention != 'to' and convention != 'from':
```

Comment:
Common Python idiom for this is:
```python
elif convention not in ('to', 'from'):
```

Reply:
Ah I did this with brackets earlier and it didn't work. Need more coffee. Will update.

Reply:
Hrmm...`[]` or `{}` should all work. Mine creates a tuple, the `[]` a list, and `{}` a set. All support `x in container`. That's weird.